lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -203,7 +203,7 @@ function dosomething_global_node_insert($node) {
 /**
  * Implements hook_form_alter().
  */
-function dosomething_global_form_node_form_alter(&$form, &$form_state, $form_id) {
+function dosomething_global_form_node_form_alter(&$form, &$form_state) {
```

Comment:
Looks like we still reference it in all our other `hook_form_alter`s, so I removed that commit. Strange, couldn't remember seeing that one before.